contracts/account/manager/src/commands.rs

```diff
@@ -430,34 +433,40 @@ pub fn update_info(
     Ok(ManagerResponse::action("update_info"))
 }
 
-pub fn update_subscription_status(
+pub fn update_suspension_status(
     deps: DepsMut,
     info: MessageInfo,
-    new_status: Subscribed,
+    suspension_status: Subscribed,
+    response: Response,
 ) -> ManagerResult {
     let config = CONFIG.load(deps.storage)?;
 
     // Only the subscription contract can load
     if let Some(sub_addr) = config.subscription_address {
         if sub_addr.eq(&info.sender) {
-            STATUS.save(deps.storage, &new_status)?;
-            return Ok(ManagerResponse::new(
-                "update_subscription_status",
-                vec![("new_status", new_status.to_string())],
-            ));
+            SUBSCRIPTION_STATUS.save(deps.storage, &suspension_status)?;
```

Comment:
so i'm still not convinced that we should have this be related to "subscriptions". Can we just store it as SUSPENTION_STATUS?